### PR TITLE
Typos error

### PR DESCRIPTION
**Pull request by mafiya69**
Some spelling mistakes:
Sublcasses -> Subclasses

### CODE REVIEW

**Comment by shivamvats** [comment]
@mafiya69 Thanks for the fix!
+1

File: src/basic.h [hunk 1]
```diff
@@ -270,4 +270,3 @@ const static TypeID type_code_id = ID; \
 virtual TypeID get_type_code() const { return type_code_id; };
 
 #endif
-
```
Comment (sushant-hiray) [inline src/basic.h]: @mafiya69 You accidentally deleted this line. Can you please add the new line again.

Comment (shivamvats) [inline src/basic.h]: @sushant-hiray Do we really need trailing blank lines?

Comment (certik) [inline src/basic.h]: I don't think we need trailing blank lines.

**Comment by certik** [comment]
Thanks @mafiya69 for the fix.

**Comment by mafiya69** [comment]
As some of you were asking to add a blank line. Done!

**Comment by Sumith1896** [comment]
Doesn't matter, sooner or later we will implement tests for code quality where these things will be cleared out. Thanks for the extra effort though.

**Comment by Sumith1896** [comment]
+1 to merge

**Pull request merged**